macros/base/start_print.cfg

```diff
@@ -120,19 +118,15 @@ gcode:
     # 4 ----- GANTRY LEVELING ----------------------------------
     # Do a QGL, Z_TILT_ADJUST, or nothing depending of the machine configuration
     # The correct operation is automatically selected in the TILT_CALIBRATION macro
-    {% if status_leds_enabled %}
-        STATUS_LEDS COLOR="LEVELING"
-    {% endif %}
```

Comment:
Can you put this test back ? it's mandatory to do it here...

As the file `status_leds.cfg` **is optionally included**, it's necessary to do the test **before** calling `STATUS_LEDS COLOR=xxx` to avoid an error about a non-existing macro.

Reply:
Ok done